virtcontainers/qemu.go

```diff
@@ -585,6 +584,85 @@ func (q *qemu) vhostFSSocketPath(id string) (string, error) {
 	return utils.BuildSocketPath(store.RunVMStoragePath, id, vhostFSSocket)
 }
 
+func (q *qemu) setupVirtiofsd(timeout int) (remain int, err error) {
```

Comment:
What if `timeout < 0`?

Reply:
`timeout` is internal and never changed before reaching here. I'm not against validating it but what is the place to do the validation along the call stack? That said, we do make sure it is non-negative when changing it below.